pkg/commands/git_commands/bisect.go

```diff
@@ -98,13 +97,15 @@ func (self *BisectCommands) GetInfo() *BisectInfo {
 }
 
 func (self *BisectCommands) Reset() error {
-	return self.cmd.New("git bisect reset").StreamOutput().Run()
+	cmdStr := NewGitCmd("bisect").Arg("reset").ToString()
+
+	return self.cmd.New(cmdStr).StreamOutput().Run()
```

Comment:
I was also going to vote for using the new pattern everywhere for consistency, but seeing this I'm not so sure... I find the old one a lot easier to read in simple cases like this.